app/kafka/producer.js

```diff
@@ -0,0 +1,20 @@
+const { Kafka } = require('kafkajs');
+
+const kafka = new Kafka({
+	clientId: 'graphql-schema-registry-server',
+	brokers: ['gql-schema-registry-kafka:9092'],
```

Comment:
These need to be configureable with env variables. Defaults should be set in docker-compose